jwst/regtest/test_miri_lrs_slitless_tso1.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from astropy.io.fits.diff import FITSDiff
+
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+from jwst.stpipe import Step
+
+
+DATASET_ID = "det_image_MIRIMAGE_P750Lexp1"
+
+
+@pytest.fixture(scope="module")
+def run_tso1_pipeline(jail, rtdata_module):
+    """Run the calwebb_tso1 pipeline on a MIRI LRS slitless exposure."""
+    rtdata = rtdata_module
+    collect_pipeline_cfgs("config")
+    rtdata.get_data(f"miri/lrs/{DATASET_ID}.fits")
+
+    args = [
+        "config/calwebb_tso1.cfg",
+        rtdata.input,
+        "--steps.dq_init.save_results=True",
+        "--steps.saturation.save_results=True",
+        "--steps.rscd.save_results=True",
+        "--steps.linearity.save_results=True",
+        "--steps.dark_current.save_results=True",
+        "--steps.refpix.save_results=True",
+        "--steps.jump.rejection_threshold=25",
+        "--steps.jump.save_results=True"
```

Comment:
What about the results of the ``firstframe`` and ``lastframe`` steps? Those could be included too.

Reply:
@hbushouse I do not think the first or last frame is done for MIRI tso1.  I looked at the existing  cal webb_tso1.cfg and at the document K Gordon wrote https://outerspace.stsci.edu/display/JWSTCC/CALWEBB_DETECTOR1
and it looks like both steps are skipped.